pootle/apps/pootle_store/unit/proxy.py

```diff
@@ -39,3 +40,18 @@ def hasplural(self):
             self.source is not None
             and (len(self.source.strings) > 1
                  or getattr(self.source, "plural", None)))
+
+    def getnotes(self, origin=None):
+        return self.unit["%s_comment" % origin]
+
+    def getcontext(self):
```

Comment:
fair point - im wondering if its useful outside of diff - here its used in the update too - and what we should call it